sauce/features/accounts/customFlagNames/settings.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  name: 'CustomFlagNames',
+  type: 'checkbox',
+  default: false,
+  section: 'accounts',
+  title: 'Add Custom Flag Names',
+  description: 'Adds the ability to set custom flag names.'
```

Comment:
I'd  add text that lets the user know the custom flag names are local to the computer they set them on. You might consider using markdown to emphasize this point. I suspect we may get issues opened anyway because people think the  values should be in all their YNAB "sessions". 

I'd also add text that says the custom flag name will only be visible when the cursor is over the flag.